components/tools/Toggle.tsx

```diff
@@ -0,0 +1,50 @@
+import { twMerge } from 'tailwind-merge';
+
+export interface ToggleProps {
```

Comment:
export this interface from types folder.

Reply:
Done